apps/opencs/view/world/util.cpp

```diff
@@ -179,7 +181,9 @@ QWidget *CSVWorld::CommandDelegate::createEditor (QWidget *parent, const QStyleO
     // (the third parameter of ColorEditor's constructor)
     else if (display == CSMWorld::ColumnBase::Display_Colour)
     {
-        return new CSVWidget::ColorEditor(index.data().value<QColor>(), parent, true);
+        int colorInt = index.data().toInt();
+        QColor color = QColor(colorInt & 0xff, (colorInt >> 8) & 0xff, (colorInt >> 16) & 0xff);
```

Comment:
This line of code is repeated rather often. Wouldn't it be better to move it to the ColorEditor class? An overlaoded constructor, maybe?